public/app/features/browse-dashboards/components/BrowseActions/DeleteModal.tsx

```diff
@@ -17,6 +18,13 @@ export interface Props {
 
 export const DeleteModal = ({ onConfirm, onDismiss, selectedItems, ...props }: Props) => {
   const [isDeleting, setIsDeleting] = useState(false);
+  const confirmButtonText = useMemo(
+    () =>
+      isDeleting
+        ? t('browse-dashboards.action.deleting', 'Deleting...')
+        : t('browse-dashboards.action.delete-button', 'Delete'),
+    [isDeleting]
+  );
```

Comment:
I don't think the useMemo hook is necessary in this case, you can just use the t() function inside the ternary directly